packages/v4/src/content/design-guidelines/components/notification-drawer/notification-drawer.md

```diff
@@ -55,7 +53,7 @@ If you want to sort notifications by category, collapsible headers may be used t
 
 <img src="./img/Notification-drawer-in-page.png" alt="notification drawer placement" width="825"/>
 
-**1. Notification badge:** The blue dot indicates to users that there are unread notifications in the drawer. Clicking on the icon will toggle the drawer open or closed.
+**1. Notification badge:** Whenever the notification drawer is used, a [Notification badge](/components/notification-badge) must be placed in the masthead. Clicking on the badge will toggle the drawer open or closed. Depending on options supported in the application, the badge can also be used to display the number of notifications in the drawer and whether there are unread or critical notifications needing attention. See the [Notification badge](components/notification-badge) usage guidelines for more information about these options.
```

Comment:
These 3 bullets are still not indented, and neither are the ones under "placement in page"!

Reply:
Also, missing a / on the second notification badge link!